src/commands/report.ts

```diff
@@ -54,3 +81,87 @@ export async function execute(interaction: CommandInteraction) {
     });
   }
 }
+
+function generateModal(user: any, options: any) {
```

Comment:
I think we can type this a bit better. Let me know if you want some help with that, but basically you can hover over the value that's being passed in and see what type it is

Reply:
Yeah I might need help with this, I wasn't able to just infer the type, IDE was not happy

Reply:
Attempt 1: https://github.com/chainpatrol/discord-bot/pull/23/commits/6b9914c106d5cda1007acffe224728ddb84626e1